test/api/v01/vrp_test.rb

```diff
@@ -20,7 +20,12 @@
 
 class Api::V01::VrpTest < Api::V01::RequestHelper
   include Rack::Test::Methods
-  include FakeRedis
+
+  def clear_optim_redis_count
```

Comment:
it creates a new test called `clear_optim_redis_count` is'n it? Is it what we want if it is the case?